src/main/java/me/drex/itsours/claim/permission/PermissionManager.java

```diff
@@ -19,21 +19,28 @@ public PermissionManager(CompoundTag tag) {
     }
 
     private void fromNBT(CompoundTag tag) {
```

Comment:
camelCase? `fromNbt`

Reply:
nbt means **N**amed **B**inary **T**ag and therefor it should in my opinion stay liked that

Reply:
Well I would like to follow Yarn's naming schemes and NBT is named Nbt